message.json

```diff
@@ -1,18 +1,20 @@
 {
-  "RequestType": "${request_type}",
-  "ServiceToken": "${bridgecrew_sns_topic}",
   "PhysicalResourceId": "",
-  "Source": "Terraform",
+  "RequestType": "${request_type}",
   "ResourceProperties": {
-    "ServiceToken": "${bridgecrew_sns_topic}",
+    "AWSAccountId": "${account_id}",
     "Account": "bridgecrew",
     "ActionType": "AWSConfigActivation",
-    "AWSAccountId": "${account_id}",
+    "ApiToken": "${api_token}",
+    "CrossAccountRoleArn": "${role_arn}",
+    "CustomerName": "${customer_name}",
     "ExternalId": "${external_id}",
+    "IntegrationName": "Terraform",
+    "ServiceToken": "${bridgecrew_sns_topic}",
     "TemplateVersion": "1.5",
     "Type": "AWS_CT_CFG",
-    "CrossAccountRoleArn": "${role_arn}",
-    "CustomerName": "${customer_name}",
-    "IntegrationName": "Terraform"
-  }
+    "Version": "1.O"
```

Comment:
Why do we need this and the template version together?

Reply:
its gone